tests/browser-automated-tests-playwright/tests/test_order_plan_with_subscription_and_upfront_charge.js

```diff
@@ -70,14 +70,20 @@ async function test_order_plan_with_subscription_and_upfront_charge(browsers, br
     await new Promise(x => setTimeout(x, 15000)); //15 secconds
     await page.goto(PLAYWRIGHT_HOST + 'admin/subscribers')
     await page.screenshot({ path: `view-subscribers-${browserType}.png` });
-
     // Verify that subscriber is present in the list
     const subscriber_email_content = await page.textContent('.subscriber-email');
     assert(subscriber_email_content === 'john@example.com');
 
     // Verify that plan is attached to subscriber
     const subscriber_subscription_title_content = await page.textContent('.subscription-title');
     assert(subscriber_subscription_title_content === 'Hair Gel');
+    
+    // Click Refresh Subscription
+    await page.click('#refresh_subscriptions'); // this is the refresh subscription
+    await page.textContent('.alert-heading') === "Notification";
```

Comment:
Same , but checking for both up-front and recurring price https://github.com/Subscribie/subscribie/pull/573/files#r643543510